Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -294,21 +356,33 @@ class AccountStatusSetting: WithAccountSetting {
                 if let url = try? cs?.asURL() {
                     viewController.url = url
                 }
-                
+                break
```

Comment:
`switch` statements in Swift do not implicitly fall through, so you don't need `break`.

https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/ControlFlow.html#//apple_ref/doc/uid/TP40014097-CH9-ID130